UPGRADING.md

```diff
@@ -1,5 +1,53 @@
 # Upgrade guidelines
 
+## 0.64.0 to 0.66.0
```

Comment:
Would be 0.67.0 if part but maybe a version placeholder makes more sense.

Reply:
Changed it to `$CURRENT_VERSION to $NEXT_VERSION`. Same in NOTES.txt